qemu/tests/multi_disk_wild_hotplug.py

```diff
@@ -0,0 +1,123 @@
+"""Wild hot-plug-unplug test"""
+import copy
+import logging
+import time
+
+from avocado.utils import process
+from virttest.qemu_monitor import QMPCmdError
+from virttest import error_context, env_process
+
+
+# This decorator makes the test function aware of context strings
+@error_context.context_aware
+def run(test, params, env):
+    """
+    1) Boot the vm with multiple data disks.
+    2) Run some operation in guest if request.
+    3) Execute device_del for data disks.
+    4) Sleep some time.
+    5) Execute device_add for data disks.
+    6) Sleep some time.
+    7) repeat step 3-6 if request.
+
+    :param test: QEMU test object.
+    :param params: Dictionary with the test parameters.
+    :param env: Dictionary with test environment.
+    """
+
+    def _run_sg_luns():
+        script_cmd = r"""cat << EOF >/tmp/guest-run.sh
+trap 'kill \$(jobs -p)' EXIT SIGINT
+
+for i in \`seq 0 32\` ; do
+    while true ; do
+        sg_luns /dev/sdb > /dev/null 2>&1
```

Comment:
`/dev/sdb` is correct when the` driver_format = virtio_blk`,
I think there is no `/dev/sd*` inside guest, just `/dev/vd*`.

Reply:
This script only run for virtio_scsi.